src/GetAPI.php

```diff
@@ -2,7 +2,6 @@
 
```

Comment:
Isn't this a separate task branch?

Reply:
yeees we branched off the task branch :/